pkg/trident/svm.go

```diff
@@ -174,32 +175,42 @@ func (m *SvnManager) CreateSVM(ctx context.Context, opts CreateSVMOptions) error
 	return nil
 }
 
-// getFirstNodeInCluster fetches the first node name found in the ONTAP cluster
+// getAllNodesInCluster fetches the first node name found in the ONTAP cluster
 // Needs to be changed, waiting for Netapp answer
-func (m *SvnManager) getFirstNodeInCluster(ctx context.Context) (string, error) {
+func (m *SvnManager) getAllNodesInCluster(ctx context.Context) ([]string, error) {
 	m.log.Info("Fetching first available node in cluster...")
 
+	var nodeUUIDs []string
 	params := cluster.NewNodesGetParamsWithContext(ctx)
 	params.SetFields([]string{"uuid", "name"})
 
 	result, err := m.ontapClient.Cluster.NodesGet(params, nil)
 	if err != nil {
-		return "", fmt.Errorf("failed to fetch nodes: %w", err)
+		return nil, fmt.Errorf("failed to fetch nodes: %w", err)
 	}
 	for _, node := range result.Payload.NodeResponseInlineRecords {
 		m.log.Info("Node in Response", "nodes", node)
 	}
 
 	if result.Payload == nil {
-		return "", errors.New("no node information returned")
+		return nil, errors.New("no node information returned")
 	}
 
 	if len(result.Payload.NodeResponseInlineRecords) == 0 {
-		return "", fmt.Errorf("nodeResponseInlineRecords is empty")
+		return nil, fmt.Errorf("nodeResponseInlineRecords is empty")
+	}
+	nodeRecords := result.Payload.NodeResponseInlineRecords
+	for _, node := range nodeRecords {
+		nodeUUIDs = append(nodeUUIDs, node.UUID.String())
+	}
+
+	if len(nodeRecords) < 2 {
```

Comment:
This would break the test setup as we do not have the chance to have 2 nodes there.